microsoft-edge/webview2/how-to/debug-visual-studio.md

```diff
@@ -6,98 +6,181 @@ ms.author: msedgedevrel
 
+* To debug scripts, the app must be launched from within Visual Studio.
 
-To debug your code, first install and set up the script debugger tools in Visual Studio, per the following two sections.
+* You cannot attach a debugger to a running WebView2 process.
 
 
 <!-- ====================================================================== -->
 ## Install the JavaScript diagnostics component
 
-First, install the **JavaScript diagnostics** component in **Desktop development with C++**, as follows.
+Use the **Visual Studio Installer** to install the **Desktop development with C++** workload of Visual Studio, including its **JavaScript diagnostics** component, as follows.  Or, if you are using a language workload other than C++, select that workload's **JavaScript diagnostics** component, similar to the below steps.
+
+1. If not done already, clone the WebView2Samples repo, as described in [Clone the WebView2Samples repo](./machine-setup.md#clone-the-webview2samples-repo) in _Set up your Dev environment for WebView2_.
+
+1. If not done already, install Visual Studio 2022, as described in [Install Visual Studio](./machine-setup.md#install-visual-studio) in _Set up your Dev environment for WebView2_.
+
+1. In Visual Studio 2022, install the **Desktop development with C++** workload, as follows:
 
 1. In the Windows Explorer bar, type `Visual Studio Installer`.
 
 1. Select **Visual Studio Installer** to open it.
 
 1. In the Visual Studio Installer, on the installed version, click the **More** button, and then select **Modify**.
 
-1. In Visual Studio, under **Workloads**, select the **Desktop Development with C++** setting:
+1. In Visual Studio, under **Workloads**, select the **Desktop Development with C++** workload, so that a checkmark appears:
 
    ![Visual Studio Modifying Workloads Screen](./debug-visual-studio-images/workloads.png)
 
 1. Select **Individual components**, at the top.
 
 1. In the search box, enter `JavaScript diagnostics`.
 
-1. Select the **JavaScript diagnostics** setting.
-
-1. Click **Modify**.
+1. Select the **JavaScript diagnostics** checkbox:
 
    ![Visual Studio: Modifying values in the 'Individual components' tab](./debug-visual-studio-images/indiv-comp.png)
 
+1. Click the **Modify** button.
+
+Continue with [Enable script debugging for WebView2 apps](#enable-script-debugging-for-webview2-apps), below.
+
 
 <!-- ====================================================================== -->
 ## Enable script debugging for WebView2 apps
 
-Second, enable script debugging for WebView2 apps, as follows.
+Second, enable script debugging for WebView2 apps, as follows.  The following steps walk you through:
+* Opening the Win32APISample sample in Visual Studio 2022.
+* Working within your own working branch of the WebView2Samples repo, where some of the project files are modified during updating to Visual Studio 2022.
```

Comment:
where did the "create branch" steps go?  check commits, & check PR about main sample.

"The following steps walk you through ... Working within your own working branch of the WebView2Samples repo, where some of the project files are modified during updating to Visual Studio 2022."

Reply:
removed mention of branch; leveraged following main-sample article, stated expected setup state